CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/ColorInputPane.java

```diff
@@ -136,6 +133,13 @@ public ColorInputPane(final PluginParameter<ColorParameterValue> parameter) {
         });
 
         getChildren().add(hbox);
+        if (recentValues != null) {
+            if (recentValues.size() > 1) {
+            parameter.setStringValue(recentValues.get(1));
+         } else {
+            parameter.setStringValue(recentValues.get(0));
+            }
```

Comment:
Can change this if-else to tertiary statement: `parameter.setStringValue(recentValues.get(recentValues.size() > 1 ? 1 : 0))`